PlexRequests.UI/Modules/RequestsBetaModule.cs

```diff
@@ -186,6 +186,37 @@ private Negotiator LoadRequests()
 
             }
 
+            IEnumerable<RootFolderModel> rootFolders = new List<RootFolderModel>();
```

Comment:
This module is not being used yet just a FYI, there is no code being used in this class